components/volume/store/errors.go

```diff
@@ -16,6 +16,13 @@ var (
 	errNameConflict = errors.New("conflict: volume name must be unique")
 )
 
+// OperationErr is an error used by the store when operations fail
```

Comment:
I tried that, but I ended up with a bunch of test failures on `if err != nil`. I think it has something to do with go interfaces.